cob_elmo_homing/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_elmo_homing)
+add_compile_options(-std=c++11)
```

Comment:
Wrong file?

Reply:
And remove it here, I cannot guarantee that it works properly with C++1 mixed in for kinetic